Packs/CommonScripts/Scripts/FormatURL/FormatURL.py

```diff
@@ -442,10 +442,8 @@ def check_valid_character(self, index: int) -> tuple[int, str]:
                 return len(self.modified_url), part
 
         elif char == '\\':
-            # Edge case of the url ending with quotes and an escape char before them
-
-            if index + 1 == len(self.modified_url) or self.modified_url[index + 1] == "\"":
-                return len(self.modified_url), part
+            # Edge case of the url ending with an escape char
```

Comment:
correct me if I'm wrong, now it goes to the return statement even if it's not the end of the string. how does this resolve the issue?

Reply:
As forward slash is invalid it will return everything caught up to that moment and regard it as the URL. But, this is a safety check as forward slash is no longer caught by the regex.